### PR TITLE
🍰 Get Cypress Tests Running Again

### PR DESCRIPTION
**Pull request by ulfgebhardt**
### Issues

- None

### Todo

- [ ] Post Image Crop
- [ ] Terms & Conditions
- [ ] Registration
- [ ] Remove Random Amazon Images
- [ ] Password Forgotten
- [ ] Editor
- [ ] Toast Messages unify
- [ ] Navigation & Menus
- [ ] HidePost (actual hiding)
- [ ] Error beim Speichern von nur einem Link (Vimeo Post)
- [ ] Remove cucumber backend behaviour tests
- [ ] Search.feature GraphQL Error
- [ ] Toaster Error Handling in Console
- [x] Cypress remove setup command
- [x] Cypress adjust documentation

### CODE REVIEW

**Comment by Tirokk** [comment]
Hey @ulfgebhardt 
that’s really great that your cared and got it a least partly running. ❤️🍀

**Comment by Tirokk** [comment]
A reminder for me to test the search feature … @Tirokk

**Review by Tirokk — CHANGES_REQUESTED** [review]
Hey that’s really supercool @ulfgebhardt  !!!!
Great that we have the Cypress tests back. 🚀🚀🚀💫💫💫

I found problems:

- If I run `yarn cypress:open` and run all the tests of the list I get the error:

Bildschirmfoto 2021-04-14 um 13 49 26

- If I run `yarn cypress:run` I get timeout errors like:

Bildschirmfoto 2021-04-14 um 14 03 02

**Comment by ulfgebhardt** [comment]
@Tirokk I fixed the documentation & removed the used commands, but the error you post is a thing of your machine I believe. As I understand it it is only happening if you run all tests together, which is quite memory intensive. If running the tests single works it should be fine.

File: cypress/cypress.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "projectId": "qa7fe2",
   "ignoreTestFiles": "*.js",
   "baseUrl": "http://localhost:3000",
```
Comment (Tirokk) [inline cypress/cypress.json:4]: Add the line

```json
  "chromeWebSecurity": false,
```

here?

**Review by Tirokk — COMMENTED** [review]
Nice that you changed the docs as well @ulfgebhardt ❤️

I understand that you liked to add the line I need for running the tests in Chromium as well.
See below …

**Review by Tirokk — APPROVED** [review]

File: cypress/cypress.json (hunk 1 above)
Comment (ulfgebhardt) [reply cypress/cypress.json:4]: Please commit it @Tirokk <3

**Comment by Tirokk** [comment]
I pushed the Chromium security fix @ulfgebhardt 
You can merge. ;-)

**Pull request merged**